examples/ciao/group_vars/all

```diff
@@ -1,20 +1,28 @@
 ---
+# Vars required for mariadb and os-common
 database_root_password: secret
 
+# Vars required for os-common
 keystone_fqdn: identity.example.com
 keystone_admin_password: secret
+
+# Vars required for os-keystone
 keystone_database_password: secret
 keystone_root_domain: example.com
 keystone_p12password: secret
 
+# Vars required for ciao-common
 ciao_controller_fqdn: controller.intel.com
+
+# Vars required for ciao-controller
 ciao_service_user: csr
 ciao_service_password: secret
 ciao_admin_email: admin@example.com
 ciao_cert_organization: Example, Inc.
 ciao_guest_user: demouser
```

Comment:
Should consider more specific what this ``guest_user`` means?, maybe being more clear on the difference of ``ciao_service_user`` and ``ciao_guest_user``.

Someone with not much cloud-related experience will not be able to understand what's a guest user.